include/usb/class/usb_dfu.h

```diff
@@ -0,0 +1,122 @@
+/***************************************************************************
+ *
+ * Copyright(c) 2015,2016 Intel Corporation.
+ * Copyright(c) 2017 PHYTEC Messtechnik GmbH
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * * Redistributions of source code must retain the above copyright
+ * notice, this list of conditions and the following disclaimer.
+ * * Redistributions in binary form must reproduce the above copyright
+ * notice, this list of conditions and the following disclaimer in
```

Comment:
License text gets somehow screwed up with double '*' though I understand you have copied it from dfu sample, @nashif can we use general SPDX stuff here instead of full license?